conjure-java-core/src/integrationInput/java/com/palantir/product/ListExample.java

```diff
@@ -165,9 +167,9 @@ public static final class Builder {
 
         private List<@Safe String> items = new ArrayList<>();
 
-        private List<Integer> primitiveItems = new ArrayList<>();
+        private List<Integer> primitiveItems = new IntegerArrayList();
 
-        private List<Double> doubleItems = new ArrayList<>();
+        private List<Double> doubleItems = new DoubleArrayList();
```

Comment:
This is what the final codegen will look like. We should now be able to change the implementation of DoubleArrayList to use a primitive array.

Reply:
new DoubleArrayList() should be replaced with ConjureCollections factory method